### DISCUSSION TITLE
Carahunge Landscape

### DISCUSSION THREAD

**Discussion by dmonty2** [Stellarium/stellarium#1769] [Show and tell]
Had some fun playing with Carahung Stone Circle in Armenia as a series of Landscapes in Stellarium.  It is a puzzle game trying to figure out the purpose for the stones in the context of location and time.

Create a new location:
Carahunge in Armenia
N 39° 33' 2.52"
E 46° 1' 42.96"
1770 m

See README.md for some observations.  Correcting mistakes are welcome : )

[https://gitlab.com/dmonty2/stellarium_config](https://gitlab.com/dmonty2/stellarium_config)

**Comment by alex-w** [comment]
Discussion: https://www.cloudynights.com/topic/777829-carahunge-stone-circle-in-stellarium/
